man-roxygen/path.R

```diff
@@ -1,4 +1,3 @@
-#' @param path This is required The path to the new folder to create relative to
-#'   root.
+#' @param path Path in the user's Dropbox to save the file.
```

Comment:
I think it's good to fix #147, but this isn't quite right either, since many of these functions aren't doing any saving (e.g. `drop_delete`, `drop_share`).

I'm not sure how to word a catch-all param template here; maybe we need to avoid templating some or all of them, like we do for `drop_upload`?